apps/exports/models.py

```diff
@@ -175,16 +200,38 @@ def mark_skipped(self):
 
 
 class ExportRun(ExportRunBase):
-    base_export_config = models.ForeignKey(ExportConfig, on_delete=models.CASCADE, related_name='runs')
-    export_config_version = models.ForeignKey(Version, on_delete=models.CASCADE, null=True)
+    base_export_config = models.ForeignKey(
+        ExportConfig,
+        on_delete=models.CASCADE,
+        related_name='runs',
+    )
+    export_config_version = models.ForeignKey(
+        Version,
+        on_delete=models.CASCADE,
+        null=True,
+    )
 
 
 class MultiProjectExportRun(ExportRunBase):
-    base_export_config = models.ForeignKey(MultiProjectExportConfig, on_delete=models.CASCADE, related_name='runs')
-    export_config_version = models.ForeignKey(Version, on_delete=models.CASCADE, null=True)
+    base_export_config = models.ForeignKey(
+        MultiProjectExportConfig, on_delete=models.CASCADE, related_name='runs'
```

Comment:
nit: more line breaks like `export_config_version` below?

Reply:
Thank you. a38cb71